modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc

```diff
@@ -132,6 +132,14 @@ calculate accordingly.
 
 == PHP-FPM
 
+Please note that the ownCloud QA team is using solely mod_php for testing. 
+Therefore in a production environment it is recommended to do the same.
```

Comment:
Therefore in a production environment it is recommended to do the same.

It's highly recommended to use `mod_php` in your production environment for optimal performance and stability.